Automattic/wp-calypso
# Tests: Get Visual Regression tests passing in TC

#### Changes proposed in this Pull Request

* This PR gets the VR tests passing in TC. It also sets up for some configuration for number of threads that are used.
* Sets up report to be on a tab. I think that will work when merged.
* Two reference images were updated to get them to pass

#### Testing instructions

* Make sure tests pass and see that they also pass [here](https://teamcity.a8c.com/viewType.html?buildTypeId=calypso_VisualRegressionTests&branch_calypso_WPComTests=fix%2FVR-testing&tab=buildTypeStatusDiv)
